atlp-rwanda/bravo-bn-fe
# #182201672 ch-codeclimate-test-coverage: setup codeclimate test coverage

#### What does this PR do?
This PR is for maintaining the project's code coverage using codeclimate


#### How should this be manually tested?
click on the badges in the `README` file to check more details about the coverage, tests and maintenance of the project.
##run: 
```bash

`npm run test:cov`  for the coverage

```


#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201672
